sqldelight-core/src/main/kotlin/com/squareup/sqldelight/core/lang/Constants.kt

```diff
@@ -46,3 +46,5 @@ internal val TRANSACTION_TYPE = TRANSACTER_TYPE.nestedClass("Transaction")
 internal val CONNECTION_TYPE = ClassName("com.squareup.sqldelight.db", "SqlDatabaseConnection")
 internal val CONNECTION_NAME = "db"
 
+internal fun isUnchangedPropertyName(name: String) =
+  name.startsWith("is") && !name.toCharArray()[2].isLowerCase()
```

Comment:
`toCharArray()` call is redundant. http://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-char-sequence/get.html

Reply:
Also needs test. "island" or something